content/reference-rippled.md

```diff
@@ -10955,6 +10957,182 @@ The response follows the [standard format](#response-formatting), with a success
 
 * Any of the [universal error types](#universal-errors).
 
+
+
+
+# Peer Protocol #
+
+Servers in the Ripple Consensus Ledger communicate to each other using the Ripple peer protocol, also known as RTXP. Peer servers connect via HTTPS and then perform an [HTTP upgrade](https://tools.ietf.org/html/rfc7230#section-6.7) to RTXP. (For more information, see the [Overlay Network article in the `rippled` repository](https://github.com/ripple/rippled/blob/906ef761bab95f80b0a7e0cab3b4c594b226cf57/src/ripple/overlay/README.md#handshake).)
+
+## Configuring the Peer Protocol ##
+
+In order to participate in the Ripple Consensus Ledger, `rippled` servers connects to arbitrary peers using the peer protocol. (All such peers are treated as untrusted, unless they are [clustered](tutorial-rippled-setup.html#clustering) with the current server.)
+
+Ideally, the server should be able to send _and_ receive connections on the peer port. You should forward the port used for the peer protocol through your firewall to the `rippled` server. The [default `rippled` configuration file](https://github.com/ripple/rippled/blob/develop/doc/rippled-example.cfg) listens for incoming peer protocol connections on port 51235 on all network interfaces. You can change the port used by editing the appropriate stanza in your `rippled.cfg` file.
+
+Example:
+
+```
+[port_peer]
+port = 51235
+ip = 0.0.0.0
+protocol = peer
+```
+
+## Peer Crawler ##
+
+The Peer Crawler reports information about other servers a `rippled` server is connected to as peers. The [`peers` command](#peers) in the [WebSocket and JSON-RPC APIs](#websocket-and-json-rpc-apis) also returns a similar, more comprehensive set of information, but requires admin access to the server. The Peer Crawler response is available to other servers on a non-privileged basis through the Peer Protocol (RTXP) port.
+
+#### Request Format ####
+
+To request the Peer Crawler information, make the following HTTP request:
+
+* **Protocol:** https
```

Comment:
As stated in the intro to this pull request (no surprise you missed it, the link kind of skips past it):

> 

I intentionally deviated from the standard REST method template because this HTTP request for the crawl command is unique (even the peer protocol messages mostly don't work this way), so I have to clarify information that's usually put in another section that applies to all the methods in an API.